### PR TITLE
added tutor field to titlepage

### PR DESCRIPTION
**Pull request by pfeifferj**

### CODE REVIEW

**Comment by cagix** [comment]
Actually, this PR would introduce two changes: add a new metadata variable `tutor` as well as move the title logo in front of the title ...

File: eisvogel.tex [hunk 1]
```diff
@@ -846,6 +846,11 @@
 $else$
 {
   \setstretch{1.4}
+  $if(logo)$
```
Comment (cagix) [inline eisvogel.tex]: this moves the title logo in front of the author and may break other peoples workflows ...

File: eisvogel.tex [hunk 2]
```diff
@@ -854,15 +859,12 @@
   $endif$
   \vskip 2em
   \noindent {\Large \textsf{$for(author)$$author$$sep$, $endfor$}}
+  \vskip 1em
```
Comment (cagix) [inline eisvogel.tex]: @pfeifferj  could you enclose your addtion by 

```
$if(tutor)$
... // your addition
$endif$
```

this way the extra `\vskip 1em` would not be visible when not using the new variable `tutor` ...

Comment (pfeifferj) [reply eisvogel.tex]: thanks for the pointer :)

**Comment by pfeifferj** [comment]
I made the changes you suggested

File: eisvogel.tex [hunk 3]
```diff
@@ -854,15 +855,18 @@
   $endif$
   \vskip 2em
   \noindent {\Large \textsf{$for(author)$$author$$sep$, $endfor$}}
+  $if(tutor)$
+  \vskip 1em
+  \noindent {\Large \textsf{$for(tutor)$$tutor$$sep$, $endfor$}}
+  $endif$
   \vfill
 }
 $endif$
```
Comment (Wandmalfarbe) [inline eisvogel.tex]: Could you revert everything after this line? You accidentally added `$endif$<Paste>`.

**Comment by Wandmalfarbe** [comment]
I'm a bit hesitant to include it in the template because a variable `tutor` is a very specific use case. What if one wants to display an `institute`, an `affiliation` or simply another line of text? I'm currently thinking about a generic system with variables like `line-1`, `line-2` and so on that can be used to display any text. At the same time this may be to generic and complicated.

I'll need to think about it a bit more.

File: eisvogel.tex (hunk 3 above)
Comment (pfeifferj) [reply eisvogel.tex]: Sure, I got rid of the accidental `<Paste>`

**Comment by cagix** [comment]
perhaps this generic solution is indeed, ehm, more generic and would support a few more use-cases :)

however, when working on my [pandoc-thesis](https://github.com/cagix/pandoc-thesis) i wished more than once for a more modular eisvogel template,  i.e. smaller tex-snippets to be included via `-H` and `-B` into the standard pandoc template ... 

this would make upgrades to newer pandoc versions easier (regarding the eisvogel project) and would also make it easier for others to adapting eisvogel to their needs...

**Comment by np-8** [comment]
I agree with @Wandmalfarbe  that it would be better to add generic `author-line-2` or similar instead of a `tutor` field.

**Comment by francocalvo** [comment]
Has this been considerer again? I'd like to have more than one line for the authors. In my case, 'institution' would be really useful.

**Pull request closed without merging**